modules/whosOnFirst.py

```diff
@@ -0,0 +1,138 @@
+import random
```

Comment:
Am I seeing things, or did you add a second copy of the who's on first code?

Reply:
I renamed from whos_on_first to whosOnFirst but my files glitched out and whos_on_first was created again